about-peer-review/aims-and-scope.md

```diff
@@ -24,9 +24,12 @@ package.
 support open science. 
 
 ## Is Your Package in Scope For pyOpenSci Review?
-pyOpenSci reviews packages within a set of categories define below. 
-If you are unsure whether your package fits into one of these categories, please 
-open a [pre-submission inquiry via a GitHub Issue](LINK) to get feedback from 
+
+pyOpenSci only reviews packages that fall within our specified domain and 
+technical scope listed below. 
+
+If you are unsure whether your package is in scope for review, please 
+open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSci/software-review/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) to get feedback from 
```

Comment:
The template is well laid out! Would there be a legitimate interest in knowing if the submitting author is committing to the package's maintenance at this point?

Reply:
good point. @Batalex do you think we should add that to the presubmission template as a link ? we just want to  avoid people trying to publish a package (a one time activity) and be done vs vet it and maintain over time.

Reply:
I took the liberty of submitting a PR: https://github.com/pyOpenSci/software-review/pull/66

Reply:
Thanks! i'll look at that PR today as well. going to focus on this pr first and then your pr in software review! thanks @Batalex i do think it's good to set expectations early. otherwise someone would get to the review and maybe walk away then. it saves us all time to tell them early what we expect :) excellent feedback